js/app/models/simpleEventModel.js

```diff
@@ -51,7 +51,8 @@ app.factory('SimpleEvent', function () {
 		'cutype',
 		'cn',
 		'delegated-from',
-		'delegated-to'
+		'delegated-to',
+		'lang'
```

Comment:
RFC 5545 defines a [language parameter](https://tools.ietf.org/html/rfc5545#section-3.2.10) which is also [applicable to the Attendee property](https://tools.ietf.org/html/rfc5545#section-3.8.4.1).

Please use officially defined parameter instead of a custom one :)